src/components/large-pipeline-warning/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import { toggleDisplayLargeGraph } from '../../actions/graph';
+import './large-pipeline-warning.css';
+
+export const LargePipelineWarning = ({
+  nodeCount,
+  theme,
+  onToggleDisplayLargeGraph
+}) => {
+  return (
+    <div
+      className={classnames('pipeline-warning', {
+        'kui-theme--dark': theme === 'dark',
+        'kui-theme--light': theme === 'light'
+      })}>
+      <div className="pipeline-warning__title">Your pipeline is large.</div>
+      <div className="pipeline-warning__subtitle">
+        Your pipeline might take a while to render because it has{' '}
+        <b>{nodeCount}</b> nodes. Use the sidebar controls to select a smaller
+        graph, or click to render.
+      </div>
+      <button
+        className={classnames('pipeline-warning__btn', {
+          'kui-theme--dark': theme === 'dark',
+          'kui-theme--light': theme === 'light'
+        })}
+        onClick={() => onToggleDisplayLargeGraph(true)}>
+        Render it anyway
+      </button>
+    </div>
+  );
+};
```

Comment:
Adding this here because I can't comment on an image file:

The chonky goose image has been removed from this component and yet the image `src/static/images/chonky-goose.jpg` is still present in the images directory. Can you please delete it?

Reply:
just deleted the image, thanks for spotting